src/veh_interact.cpp

```diff
@@ -514,6 +514,24 @@ bool veh_interact::is_drive_conflict()
     return has_conflict;
 }
 
+bool veh_interact::can_self_jack(const vehicle * veh)
```

Comment:
Isn't `veh` a class member? (the other functions don't declare it)

Reply:
@ahmedcharles is right, the veh_interact class already has a `veh` member.  However I don't actually see a reason to put `can_self_jack` into the veh_interface class/interface to begin with since no one other than it needs the function.

I think it's better to just make this function static and not put it into the veh_interact interface to begin with.  It doesn't need it because the only thing it actually needs is the veh reference which it already has in the context the function is used.

It's probably not end of world either way, but if you keep it in the veh_interact class, I'd fix it to use the member it already has.